### PR TITLE
fix: User can't be edited by project admin (DEV-1373)

### PR DESCRIPTION
**Pull request by irinaschubert**
Issue Number: DEV-1373

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [x] Bugfix: represents bug fixes
- [ ] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [ ] Documentation: documentation changes (no production code change)
- [ ] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [ ] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala [hunk 1]
```diff
@@ -1516,8 +1525,14 @@ class UsersResponderADM(responderData: ResponderData) extends Responder(responde
...
 
       _ = if (userUpdatePayload.projects.isDefined) {
-
-            if (updatedUserADM.projects.map(_.id).sorted != userUpdatePayload.projects.get.sorted) {
+            // get the projects of the updated user
+            val projectMemebershipsAfterUpdate =
+              appActor
+                .ask(UserProjectMembershipsGetRequestADM(userIri, requestingUser))
+                .mapTo[UserProjectMembershipsGetResponseADM]
+            val projectMemberships = Await.result(projectMemebershipsAfterUpdate, 3.seconds)
```
Comment (subotic) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala:1533]: Can this be solved without blocking?

**Comment by subotic** [comment]
@irinaschubert In general, I don't really know what the problem is that this PR is solving. Is there any issue that you can point me to that is describing the problem?

**Comment by irinaschubert** [comment]
> @irinaschubert In general, I don't really know what the problem is that this PR is solving. Is there any issue that you can point me to that is describing the problem?

@subotic , yes, sorry, I made a mess with the branches and JIRA issues (in the end I wasn't able to rename the branch - but the rest should be correct now). It's a bug fix: https://dasch.atlassian.net/browse/DEV-1373.

**Review by BalduinLandolt — APPROVED** [review]

**Review by mpro7 — APPROVED** [review]

**Comment by subotic** [comment]
> > @irinaschubert In general, I don't really know what the problem is that this PR is solving. Is there any issue that you can point me to that is describing the problem?
> 
> @subotic , yes, sorry, I made a mess with the branches and JIRA issues. I was working on something else first and accidentally used the same branch to fix this bug. But apart from the branch name (which I didn't manage to change without making more mess) everything should be fine. There are just some small, non-functional changes that come from the other issue (the files related to the health route). But this PR is about a bug fix: https://dasch.atlassian.net/browse/DEV-1373.

no worries. I did my share of messy PRs. Happens to everyone ;-)

**Pull request merged**